docs/getting-started/function-triggers.md

```diff
@@ -0,0 +1,147 @@
+# Function Event Sources
+
+In the previous section, you created a basic function that could receive
+and respond to HTTP requests. Now, let's create a new function that can respond
+to CloudEvents that are sent by Knative Eventing resources.
+
+In this tutorial, you will create a simple Knative event source to generate
+CloudEvents, as well as other Knative Eventing resources. You don't need
+to know anything about Knative Eventing primitives to follow this tutorial.
+However, if you would like to learn more about Knative Eventing, see the
+[Using Knative Eventing](./getting-started-eventing.md) section of this tutorial.
+
+First, create and deploy the function.
+## Create a CloudEvent Function
+
+To create and deploy a CloudEvent function that echoes the CloudEvent it receives,
+you can use the `create` and `deploy` commands.
+
+Create the function by running the command:
+
+=== "func"
+
+    ```{ .console}
+    func create -l go -t cloudevents echo
+    ```
+
+=== "kn func"
+
+    ```{ .console }
+    kn func create -l go -t cloudevents echo
+    ```
+
+!!! Success "Expected output"
+    ```{ .console .no-copy }
+    Created go function in echo
+    ```
+
+And deploy the function with the `deploy` command from within the project directory:
+
+```{ .console}
+cd echo
+func deploy
+```
+
+!!! Success "Expected output"
+    ```{ .console .no-copy }
+      Received response
+      Context Attributes,
+        specversion: 1.0
+        type: echo
+        source: event.handler
+        id: 3ac510fc-95f8-4958-a18e-3ffbff22c842
+        time: 2022-09-23T12:45:23.981Z
+        datacontenttype: application/json; charset=utf-8
+      Data,
+        {
+          "id": "89f49b4c-c8c4-46d2-a99e-eeca44fa894b",
+          "time": "2022-09-23T12:45:18.852Z",
+          "type": "boson.fn",
+          "source": "/boson/fn",
+          "specversion": "1.0",
+          "datacontenttype": "application/json",
+          "data": {
+            "message": "Hello World"
+          }
+        }
+    ```
+
+## Create a Broker
+
+Now, let's create the Broker. A Broker is a resource that will receive events
+from the event source and forward them to other resources.
+For many of the following commands, you will use the `kn` CLI. If you don't
+already have the `kn` CLI installed, please refer to the [Knative
+Client installation instructions](https://knative.dev/docs/client/install-kn/).
```

Comment:
This should line up with the setup for the getting-started tutorial, which [requires having `kn` installed](https://knative.dev/docs/getting-started/).

Reply:
Also, less exposition -- maybe:

```suggestion
Next, we'll create a Broker to buffer events between the event source and the function.
```